packages/world/src/schemas/AddressToBytes32.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { console } from "forge-std/console.sol";
+import { IStore } from "store/IStore.sol";
+import { StoreSwitch } from "store/StoreSwitch.sol";
+import { StoreCore } from "store/StoreCore.sol";
+import { SchemaType } from "store/Types.sol";
+import { Bytes } from "store/Bytes.sol";
+import { Schema, SchemaLib } from "store/Schema.sol";
+import { PackedCounter, PackedCounterLib } from "store/PackedCounter.sol";
+
+// -- User defined schema and tableId --
+struct AddressToBytes32Schema {
```

Comment:
What's the reason for both `Address` and `AddressToBytes32`?

Reply:
`Address` represents `mapping(bytes32 => address)`, `AddressToBytes32` represents `mapping(address => bytes32)`. `Address` is the underlying schema for the `OwnerTable` (in this case only `address` can be owner of things), `AddressToBytes32` is the underlying schema for the `SystemRouteTable` to store the routeId of a given system. The purpose of the latter is to check whether a system is already registered in a World. Systems (= contracts) can only be registered once per World, because otherwise it would mess up the access mechanism based on addresses.